kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/net2/GwtNetworkServiceImpl.java

```diff
@@ -263,39 +261,4 @@ public static List<GwtWifiHotspotEntry> findWifiHotspots(String interfaceName) t
         }
     }
 
-    private static void logInterfaces(String origin, List<GwtNetInterfaceConfig> configs) {
```

Comment:
Decided to remove this as it logs all of the properties. Just in case we add other sensitive data to this in the future removing seems like the safest approach.